cluster/gce/util.sh

```diff
@@ -2925,7 +2935,7 @@ function attach-internal-master-ip() {
   echo "Setting ${name}'s aliases to '${aliases}' (added ${ip})"
   # Attach ${ip} to ${name}
   gcloud compute instances network-interfaces update "${name}" --project "${PROJECT}" --zone "${zone}" --aliases="${aliases}"
-  run-gcloud-command "${name}" "${zone}" "sudo ip route add to local ${ip}/32 dev eth0"
+  run-gcloud-command "${name}" "${zone}" 'sudo ip route add to local '${ip}'/32 dev $(ip route | grep default | awk '\''{print $5}'\'')' || true
```

Comment:
is ok not to fail if we don't install the route?
it seems it can masks things and making it difficult to troubleshoot

Reply:
this code path seems to be used for a large number of nodes with an internal master so it won't be invoked for our usual jobs. So as long as we are "correct" in computing eth0, we should run with it.